openrtb.go

```diff
@@ -211,6 +211,7 @@ type Geo struct {
 type User struct {
 	ID         string          `json:"id,omitempty"`         // Unique consumer ID of this user on the exchange
 	BuyerUID   string          `json:"buyeruid,omitempty"`   // Buyer-specific ID for the user as mapped by the exchange for the buyer. At least one of buyeruid or id is recommended.
+	BuyerID    string          `json:"buyerid,omitempty"`    // Buyer-specific ID for the user as mapped by the exchange for the buyer. At least one of buyeruid or id is recommended.
```

Comment:
Thanks so much, on little thing, could you please mention in the versions in the comments? Cheers